moped-editor/src/utils/mapHelpers.js

```diff
@@ -772,18 +774,35 @@ export function useFeatureCollectionToFitBounds(
   return { fitMapToFeatureCollectionOnRender };
 }
 
+const useStyles = makeStyles(() => ({
+  layerSelectButton: {
+    position: "absolute",
+    top: "10px",
+    right: "10px",
+    zIndex: 1,
+    height: "3rem",
+    width: "205px",
+    backgroundColor: "white",
+    "&:hover": {
+      backgroundColor: "white",
+    },
+  },
+  layerSelectDropdown: {
+    width: "205px",
+  },
+}));
+
 /**
  * Custom hook that creates a layer toggle UI
  * @param {array} initialSelectedLayerNames - Array of layer names to show initially
- * @param {object} classes - Holds Material UI classnames
  * @return {UseLayerObject} Object that exposes updated array of visible layers and map UI render function
  */
 /**
  * @typedef {object} UseLayerObject
  * @property {array} visibleLayerIds - Updated list of visible map layers
  * @property {function} renderLayerSelect - Function that returns JSX for layer toggle UI
  */
-export function useLayerSelect(initialSelectedLayerNames, classes) {
+export function useLayerSelect(initialSelectedLayerNames) {
```

Comment:
Small improvement so that `useLayerSelect` is in charge of its own styles and prevent the appearance of this dropdown from diverging in the view and edit components maps.